src/components/Tooltip/Tooltip.module.css

```diff
@@ -1,72 +1,126 @@
-@import '../../design-tokens/mixins.css';
+/* stylelint-disable selector-pseudo-class-no-unknown */
```

Comment:
since we're here, probs worth configuring this rule to ignore `global`: https://stylelint.io/user-guide/rules/list/selector-pseudo-class-no-unknown/#ignorepseudoclasses-regex-string

Reply:
Will add to `stylelintrc`